ssv/spectest/tests/runner/consensus/future_decided_no_instance.go

```diff
@@ -0,0 +1,114 @@
+package consensus
+
+import (
+	"fmt"
+
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/herumi/bls-eth-go-binary/bls"
+
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// FutureDecidedNoInstance tests processing a decided msg from a larger height with no running instance
+// then returning an error and don't move to post consensus as it's not the same instance decided
+func FutureDecidedNoInstance() tests.SpecTest {
+	ks := testingutils.Testing4SharesSet()
+
+	getID := func(role types.BeaconRole) []byte {
+		ret := types.NewMsgID(testingutils.TestingSSVDomainType, testingutils.TestingValidatorPubKey[:], role)
+		return ret[:]
+	}
+
+	getDecidedMessage := func(role types.BeaconRole, height qbft.Height) *types.SSVMessage {
+		signedMsg := testingutils.TestingCommitMultiSignerMessageWithHeightAndIdentifier(
+			[]*bls.SecretKey{ks.Shares[1], ks.Shares[2], ks.Shares[3]},
+			[]types.OperatorID{1, 2, 3},
+			height,
+			getID(role),
+		)
+
+		byts, err := signedMsg.Encode()
+		if err != nil {
+			panic(err.Error())
+		}
+
+		ssvMessage := &types.SSVMessage{
+			MsgType: types.SSVConsensusMsgType,
+			MsgID:   *(*types.MessageID)(getID(role)),
+			Data:    byts,
+		}
+
+		return ssvMessage
+	}
+
+	multiSpecTest := &tests.MultiMsgProcessingSpecTest{
```

Comment:
we don't need this test for proposer?

Reply:
The proposer and blinded proposer tests are in the end of the file